lib/rubocop/cop/style/ternary_parentheses.rb

```diff
@@ -91,7 +91,7 @@ def redundant_parentheses_enabled?
         end
 
         def parenthesized?(node)
-          node.source =~ /^\(.*\)$/
+          node.type == :begin
```

Comment:
There's a predicate method on `Node` for this:

```
node.begin_type?
```

Also, I'm not entirely sure why this works. 😅 With the changed implementation, does the method name still make sense?